client/my-sites/stats/activity-log/index.jsx

```diff
@@ -193,14 +218,16 @@ class ActivityLog extends Component {
 			startDate,
 		} = this.props;
 
-		const startOfMonth = moment( startDate ).startOf( 'month' ),
-			startOfMonthMs = startOfMonth.valueOf(),
-			endOfMonthMs = moment( startDate ).endOf( 'month' ).valueOf();
-		const filteredLogs = filter( logs, obj => startOfMonthMs <= obj.ts_site && obj.ts_site <= endOfMonthMs );
-		const logsGroupedByDate = map(
+		const siteOffset = this.getSiteOffsetFunc();
+
+		const startOfMonthMs = siteOffset( moment( startDate ).startOf( 'month' ) ).valueOf();
+		const endOfMonthMs = siteOffset( moment( startDate ).endOf( 'month' ) ).valueOf();
```

Comment:
My understanding is that we need to get everything into _site_ time for comparisons. We want to find activity whose site-time is in a site-month.

If I understand correctly (and today I've found out I very likely do not 😫 ), moment will try to use local (browser) time be default. But my browser-month may not correspond with site-months, which is why I think the adjustment is necessary on all the start, end and activity times.